scripts/deep_learning/clusterSplitting/Windows.ipynb

```diff
@@ -0,0 +1,336 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "460b1fea-dbb9-48b2-8d23-a10e1b834972",
+   "metadata": {},
+   "source": [
+    "## Venusarus - Windows\n",
+    "\n",
+    "Create training files from root trees obtained from DLThreeViewClusterSplittingAlgorithm when ran in training mode \n",
+    "\n",
+    "written by Isobel Mawby (i.mawby1@lancaster.ac.uk)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "72270202-55c9-48e6-9e89-7c8b673fdff4",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-info\" style=\"font-size: 18px;\">\n",
+    "    Config\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b0dedc07-1279-402d-ac86-d74eb95731c1",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import uproot\n",
+    "import random\n",
+    "import numpy as np\n",
+    "import sys\n",
+    "\n",
+    "#sys.path.insert(0, '/home/imawby/LArMachineLearningData/scripts/deep_learning/clusterSplitting')\n",
+    "\n",
+    "import Utilities"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ae944ef6-b26d-4633-9646-bb67f8d55b62",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-info\" style=\"font-size: 18px;\">\n",
+    "    Config\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "24e75509-b3ea-4d72-a5eb-2a0d57fe862f",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "WINDOW_SIZE = 48\n",
+    "NO_OVERLAP = True # True ensures that no windows overlap"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "368228af-47c6-4496-b7f6-6fcb02e25012",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-info\" style=\"font-size: 18px;\">\n",
+    "    Open File\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "df5ab065-efc2-4360-bb67-e002bbe5cdfe",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "fileName = sys.path[0] + '/files/KalmanTraining_UVW_0.root'\n",
+    "outputFileName =  sys.path[0] + '/files/KalmanTraining_UVW_0.npz'\n",
+    "treeName = 'tree'"
```

Comment:
This may be true elsewhere, but just noticed the stylistic inconsistency here with the variable names, e.g. `fileName` rather than the more pythonic `file_name`.

Reply:
done in this file, but i will check the others too

Reply:
complete in other files too